app.go

```diff
@@ -16,7 +16,6 @@ import (
 func app(cmd *cobra.Command, args []string) {
 	var version string
 	var err error
-	defaultVersion := "17.06"
 
 	// Get version of Docker benchmark to run
 	if dockerVersion != "" {
```

Comment:
Can dockerVersion ever be an empty string now?

Reply:
./docker-bench --version ""